sky/engine/core/events/VelocityTracker.idl

```diff
@@ -0,0 +1,11 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[
+  Constructor(),
+] interface VelocityTracker {
+  void addPosition(long timeStamp, long pointerId, float x, float y);
```

Comment:
should we just have one object per pointer id, and track the IDs on the Dart side?
how do we know when to discard the data for a pointer id?